lib/vdsm/network/nmstate/bridge_opts.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2016-2022 Red Hat, Inc.
```

Comment:
This module should be probably called just options, it contains both bridge and bond options parsing.

Reply:
done